src/sections/pools/farms/position/FarmingPosition.utils.tsx

```diff
@@ -73,111 +73,130 @@ export const useAllUserDepositShare = () => {
+            valueDisplay = value.times(spotPrice.spotPrice)
+
+            if (lrna.gt(0)) {
+              valueDisplay = !lrnaSp
+                ? BN_NAN
+                : valueDisplay.plus(lrna.times(lrnaSp.data.spotPrice))
+            }
+          }
 
-        if (liquidityOutResult !== "-1" && spotPrice) {
-          valueDisplay = value.times(spotPrice.spotPrice)
+          const providedAmount = normalizeBigNumber(
+            position.data.amount,
+          ).shiftedBy(-1 * meta.decimals.toNumber() ?? DECIMAL_PLACES)
+          const providedAmountDisplay = spotPrice?.spotPrice
+            ? providedAmount.times(spotPrice.spotPrice)
+            : BN_NAN
 
-          if (lrna.gt(0)) {
-            valueDisplay = !lrnaSp
-              ? BN_NAN
-              : valueDisplay.plus(lrna.times(lrnaSp.data.spotPrice))
-          }
+          const index = position.data?.assetId.toString()
+
+          memo[index] = [
+            ...(memo[index] ?? []),
+            {
+              ...position.data,
+              depositId: positionIds
+                .find(
+                  (pos) =>
+                    pos.data?.value.toString() === position.data?.id.toString(),
+                )
+                ?.data?.depositionId.toString(),
+              value,
+              valueDisplay,
+              providedAmount,
+              providedAmountDisplay,
+              lrna,
+              symbol: meta.symbol,
+            },
+          ]
         }
 
-        const providedAmount = normalizeBigNumber(
-          position.data.amount,
-        ).shiftedBy(-1 * meta.decimals.toNumber() ?? DECIMAL_PLACES)
-        const providedAmountDisplay = spotPrice?.spotPrice
-          ? providedAmount.times(spotPrice.spotPrice)
-          : BN_NAN
-
-        const index = position.data?.assetId.toString()
-
-        memo[index] = [
-          ...(memo[index] ?? []),
-          {
-            ...position.data,
-            depositId: positionIds
-              .find(
-                (pos) =>
-                  pos.data?.value.toString() === position.data?.id.toString(),
-              )
-              ?.data?.depositionId.toString(),
-            value,
-            valueDisplay,
-            providedAmount,
-            providedAmountDisplay,
-            lrna,
-            symbol: meta.symbol,
-          },
-        ]
-      }
-
-      return memo
-    }, {} as Record<string, Array<OmnipoolPosition & { value: BN; valueDisplay: BN; lrna: BN; symbol: string; depositId: string | undefined; providedAmountDisplay: BN; providedAmount: BN }>>)
+        return memo
+      },
+      {} as Record<
+        string,
+        Array<
+          OmnipoolPosition & {
+            value: BN
+            valueDisplay: BN
+            lrna: BN
+            symbol: string
+            depositId: string | undefined
+            providedAmountDisplay: BN
+            providedAmount: BN
+          }
+        >
+      >,
```

Comment:
Reduce function is generic, it would be better to declare the result in there over aliasing 

```
positions.reduce<Record<
        string,
        Array<
          OmnipoolPosition & {
            value: BN
            valueDisplay: BN
            lrna: BN
            symbol: string
            depositId: string | undefined
            providedAmountDisplay: BN
            providedAmount: BN
          }
        >>(